### DISCUSSION TITLE
Is it possible to stream to a URL consisting of a host and a port, without path.

### DISCUSSION THREAD

**Discussion by EugeneLevakhin** [bluenviron/mediamtx#1321] [Questions]
Is it possible to stream to a URL consisting of a host and a port. For example: rtsp://localhost:8554 (without path - /somelivestream)

**Comment by aler9** [comment]
Hello, rtsp-simple-server can't work without paths, since paths allow to serve multiple streams at once.

Nonetheless you can build a custom RTSP server that doesn't make use of paths by using Go and gortsplib: https://github.com/aler9/gortsplib/blob/main/examples/server/main.go

**Comment by EugeneLevakhin** [comment]
Thank you very much for the answer.

[a comment by an automated account is not shown]
